### PR TITLE
[PLINT-15] Update VMware Tanzu Cluster Monitoring Docs

### PR DESCRIPTION
**Pull request by NouemanKHAL**
### What does this PR do? What is the motivation?

Updates the installation instructions for the VMware Tanzu Cluster Monitoring Tile.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/integrations/guide/cluster-monitoring-vmware-tanzu.md [hunk 1]
```diff
@@ -47,14 +45,13 @@ Datadog Cluster Monitoring for VMware Tanzu includes the following key features:
...
-3. Go to the Tanzu Ops Manager installation dashboard and click **Import a Product** to upload the product file.
-4. Click **Import a Product** to upload the product file.
-5. Select the product file downloaded in step **2**. This adds the tile to your staging area.
-6. Click the newly added **Datadog Cluster Monitoring for VMware Tanzu** tile.
-7. Enter your [Datadog API key][3] in the **Datadog Config** section. Leave the Datadog API URL unchanged, unless directed otherwise by [Datadog Support][2].
-8. Create a UAA client account for Datadog using the [UAA CLI][12]. The Firehose Nozzle requires access to the Loggregator Firehose.
+1. Download the **Datadog Cluster Monitoring for VMware Tanzu** product file from [Pivotal Network][11].
+1. Go to the Tanzu Ops Manager installation dashboard and click **Import a Product** to upload the product file.
+1. Click **Import a Product** to upload the product file.
+1. Select the product file downloaded in step **2**. This adds the tile to your staging area.
```
Comment (maycmlee) [inline content/en/integrations/guide/cluster-monitoring-vmware-tanzu.md:51]:
```suggestion
1. Select the product file downloaded in step **1**. This adds the tile to your staging area.
```

File: content/en/integrations/guide/cluster-monitoring-vmware-tanzu.md [hunk 2]
```diff
@@ -65,13 +62,13 @@ Datadog Cluster Monitoring for VMware Tanzu includes the following key features:
...
+1. The **Datadog Firehose Nozzle Config** section contains optional configurations for the Nozzle, and the **Datadog Agent Config** section contains optional configurations for the Agent. You do not need to configure anything in either section.
     <p class='note'><strong>Note:</strong> If you are using a single Datadog account to monitor multiple foundations, you must check the <strong>Use UUID Hostname</strong> checkbox.</p>
-12. The **Datadog Cluster Agent Settings** section contains configurations for the [Datadog Cluster Agent][15] that provides autodiscovery of integrations and application container features.
-
-13. Return to the Tanzu Ops Manager Installation Dashboard and click **Apply Changes** to install Datadog Cluster Monitoring for the VMware Tanzu tile.
+1. The **Datadog Cluster Agent Settings** section contains configurations for the [Datadog Cluster Agent][15] that provides autodiscovery of integrations and application container features.
+Enter an **Authentication token** in the **Datadog Cluster Agent Settings**, a string of 32 or more characters, this token is shared by both the Cluster Agent and the Datadog Agents to secure communication.
```
Comment (maycmlee) [inline content/en/integrations/guide/cluster-monitoring-vmware-tanzu.md:70]:
```suggestion
Enter an **Authentication token** in the **Datadog Cluster Agent Settings**, a string of 32 or more characters. This token is shared by both the Cluster Agent and the Datadog Agents to secure communication.
```

**Review by maycmlee — COMMENTED** [review]
Just a couple of small suggestions! Is this good to merge after they have been addressed?

**Comment by NouemanKHAL** [comment]
Thanks! @maycmlee, the PR is ready for merge 🙇

**Pull request merged**